vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
The problem is, that the rails-assets one has a dependency on react from the rails-assets source. We want to use the react-rails gem, though, since it integrates better with Rails. Throwing the rails-assets-react-chosen out of the asset pipeline's load path was also not possible, since somehow it is not in there in the first place...